File: playground/tip.js
var tip = function (service_rating, total) {
  var t, percentage;

  if (service_rating === "Great") {
    percentage = 0.25;
  } else if (service_rating === "Good") {
    percentage = 0.2;
  } else if (service_rating === "Average") {
    percentage = 0.15;
  } else if (service_rating === "Bad") {
    percentage = 0.1;
  } else if (service_rating === "Terrible") {
    percentage = 0.05;
  } else {
    t = 0;
  }
  t = parseFloat(total) * percentage;
  return t;
}

cconsole.log(answer)

File: playground/tip.py
class Tip:
    tip_map = {
                "great":    .25,
                "good":     .2,
                "average":  .15,
                "bad":      .1,
                "terrible": .05
              }

    def __init__(self,check):
        self.check = check


    def tip_amount(self, service_rating):
        return self.check * Tip.tip_map[service_rating]

t = Tip(45.88)
print(t.tip_amount("great"))
print(t.tip_amount("average"))
print(t.tip_amount("bad"))
print(t.tip_amount("terrible"))
